reco_utils/docker/pyspark/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+ARG BASE_CONTAINER=jupyter/pyspark-notebook
+FROM $BASE_CONTAINER
+
+LABEL maintainer="Microsoft Recommender Project <RecoDevTeam@service.microsoft.com>"
+
+ARG WORK_PATH=/home/$NB_USER/work
+ARG RECO_PATH=$WORK_PATH/Recommenders
+
+USER root
+
+# Clone repo
+RUN git clone https://github.com/Microsoft/Recommenders.git && \
+    cp -rf Recommenders $WORK_PATH 
+
+# Install the needed packages
+RUN conda install --quiet --yes \
```

Comment:
I think we should have a way to add the libraries programmatically, I've seen that other libs use scripts: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/Dockerfile.gpu#L24. We could use python to get the libs

Reply:
It is a good idea for better management.

Reply:
You can also add a juptyer_notebook_config.py for default configurations

Reply:
it would be great to adjust the conda file generator script to accept a docker flag that would handle the adjusting the yaml file as needed for these cases.

Reply:
@gramhagen having some issues installing the conda dependencies re AML but agree this is a desirable approach.